packages/shared/src/components/post/PostItemCard.tsx

```diff
@@ -24,7 +28,8 @@ export interface PostItemCardProps {
   clickable?: boolean;
   onHide?: (params: HidePostItemCardProps) => Promise<unknown>;
   onContextMenu?: (event: React.MouseEvent, post: PostItem) => void;
-  isFeedbackEnabled?: boolean;
+  showVoteActions?: boolean;
+  analyticsOrigin?: Origin;
```

Comment:
I think we generally just call if origin globally.

Reply:
`origin` is tricky due to `window.origin`

Reply:
But it's not used as invocation right, prefer if we can keep it uniform for now.

Reply:
We already have `analyticsOrigin` in bunch of places, that's why I also used it. Should be good I think.